src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    UserDao newUserDao = new UserDaoImpl();
+    BetDao betDao = new BetDaoImpl();
+
+    public void handleUsers() {
+        try (Scanner scanner = new Scanner(System.in)) {
+            while (true) {
+                System.out.println("To add new USER please enter 'name login password'"
+                        + " or 'q' if you don't want to do it:");
+                String action = scanner.nextLine();
+                if (action.equalsIgnoreCase("q")) {
+                    return;
+                }
+                User newUser = null;
+                try {
+                    String[] dataStringsUser = action.split(" ");
+                    String login = dataStringsUser[0];
+                    String name = dataStringsUser[1];
+                    String password = dataStringsUser[2];
+                    newUser = new User(name, login, password);
+                    addBets(scanner, newUser);
+                } catch (NumberFormatException | IndexOutOfBoundsException e) {
```

Comment:
do we need to catch NumberFormatException here?

Reply:
Nope :( copy paste from addBets method